web/src/lib/components/pages.tsx

```diff
@@ -125,8 +125,15 @@ export default class Pages extends Component<PagesProps, PagesState> {
     }
   }
 
+  /**
+   * If the iOS app is installed, open it. Otherwise, open the App Store.
+   */
   private openInApp() {
-    window.location.href = getItunesURL();
+    window.location.href = 'commonvoice://';
+
+    setTimeout(function() {
+      window.location.href = getItunesURL();
+    }, 500);
```

Comment:
Most browsers suspend script execution when the page is not focused, so the timeout will stop running and continue if the user returns to the page. The custom link will just open the CV app. I'm not sure if it will always execute in time, but it would be surprising if it took more than 500 ms.